### ISSUE TITLE
TestTextFilterProcessor

### ISSUE THREAD

**Issue by bossenti** [apache/streampipes#2774]

**Comment by pambrocio** [comment]
please assign this to me

**Comment by pambrocio** [comment]
Build successful
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Apache StreamPipes 0.95.0-SNAPSHOT:
[INFO] 
[INFO] Apache StreamPipes ................................. SUCCESS [  1.902 s]
[INFO] streampipes-archetype-extensions-jvm ............... SUCCESS [  0.735 s]
[INFO] streampipes-archetype-pe-sinks-flink ............... SUCCESS [  0.059 s]
[INFO] streampipes-archetype-pe-processors-flink .......... SUCCESS [  0.091 s]
[INFO] StreamPipes Commons ................................ SUCCESS [  3.270 s]
[INFO] streampipes-model-shared ........................... SUCCESS [  1.046 s]
[INFO] streampipes-vocabulary ............................. SUCCESS [  1.584 s]
[INFO] streampipes-model .................................. SUCCESS [  4.995 s]
[INFO] streampipes-dataformat ............................. SUCCESS [  0.992 s]
[INFO] streampipes-messaging .............................. SUCCESS [  1.186 s]
[INFO] streampipes-client-api ............................. SUCCESS [  1.232 s]
[INFO] streampipes-dataformat-json ........................ SUCCESS [  0.978 s]
[INFO] streampipes-dataformat-cbor ........................ SUCCESS [  0.995 s]
[INFO] streampipes-dataformat-fst ......................... SUCCESS [  1.151 s]
[INFO] streampipes-security-jwt ........................... SUCCESS [  1.175 s]
[INFO] streampipes-model-client ........................... SUCCESS [  0.775 s]
[INFO] streampipes-extensions-api ......................... SUCCESS [  1.455 s]
[INFO] streampipes-test-utils ............................. SUCCESS [  1.166 s]
[INFO] streampipes-serializers-json ....................... SUCCESS [  1.328 s]
[INFO] streampipes-client ................................. SUCCESS [  1.714 s]
[INFO] streampipes-measurement-units ...................... SUCCESS [  1.791 s]
[INFO] streampipes-sdk .................................... SUCCESS [  3.374 s]
[INFO] streampipes-connect-shared ......................... SUCCESS [  2.457 s]
[INFO] streampipes-service-discovery-api .................. SUCCESS [  1.078 s]
[INFO] streampipes-dataformat-smile ....................... SUCCESS [  1.003 s]
[INFO] streampipes-storage-api ............................ SUCCESS [  1.183 s]
[INFO] streampipes-storage-couchdb ........................ SUCCESS [  2.358 s]
[INFO] streampipes-storage-management ..................... SUCCESS [  0.940 s]
[INFO] streampipes-user-management ........................ SUCCESS [  1.832 s]
[INFO] streampipes-mail ................................... SUCCESS [  1.263 s]
[INFO] streampipes-resource-management .................... SUCCESS [  2.445 s]
[INFO] streampipes-service-discovery ...................... SUCCESS [  0.951 s]
[INFO] streampipes-pipeline-management .................... SUCCESS [  3.278 s]
[INFO] streampipes-connect-management ..................... SUCCESS [  2.769 s]
[INFO] streampipes-extensions-management .................. SUCCESS [  2.861 s]
[INFO] streampipes-data-explorer-commons .................. SUCCESS [  3.362 s]
[INFO] streampipes-data-explorer .......................... SUCCESS [  2.289 s]
[INFO] streampipes-data-export ............................ SUCCESS [  1.854 s]
[INFO] streampipes-service-base ........................... SUCCESS [  1.288 s]
[INFO] streampipes-rest-shared ............................ SUCCESS [  1.043 s]
[INFO] streampipes-rest-extensions ........................ SUCCESS [  2.062 s]
[INFO] streampipes-service-extensions ..................... SUCCESS [  1.276 s]
[INFO] Apache StreamPipes Maven Plugin .................... SUCCESS [  3.777 s]
[INFO] streampipes-extensions ............................. SUCCESS [  0.008 s]
[INFO] streampipes-connectors-mqtt ........................ SUCCESS [  1.282 s]
[INFO] streampipes-pipeline-elements-shared ............... SUCCESS [  0.759 s]
[INFO] streampipes-connect-adapters ....................... SUCCESS [  0.797 s]
[INFO] streampipes-connect-adapters-iiot .................. SUCCESS [  2.059 s]
[INFO] streampipes-wrapper ................................ SUCCESS [  1.215 s]
[INFO] StreamPipes Wrapper for Standalone Pipeline Element Implementations SUCCESS [  1.338 s]
[INFO] streampipes-connectors-influx ...................... SUCCESS [  1.164 s]
[INFO] streampipes-messaging-kafka ........................ SUCCESS [  1.022 s]
[INFO] streampipes-connectors-kafka ....................... SUCCESS [  1.162 s]
[INFO] streampipes-messaging-nats ......................... SUCCESS [  0.814 s]
[INFO] streampipes-connectors-nats ........................ SUCCESS [  0.978 s]
[INFO] streampipes-connectors-opcua ....................... SUCCESS [  2.687 s]
[INFO] streampipes-connectors-pulsar ...................... SUCCESS [  2.511 s]
[INFO] streampipes-connectors-rocketmq .................... SUCCESS [  2.403 s]
[INFO] streampipes-connectors-tubemq ...................... SUCCESS [  1.076 s]
[INFO] streampipes-processors-filters-jvm ................. SUCCESS [  3.054 s]
[INFO] streampipes-connectors-plc ......................... SUCCESS [  1.755 s]
[INFO] streampipes-messaging-mqtt ......................... SUCCESS [  0.843 s]
[INFO] streampipes-messaging-pulsar ....................... SUCCESS [  0.956 s]
[INFO] streampipes-wrapper-siddhi ......................... SUCCESS [  1.879 s]
[INFO] streampipes-processors-filters-siddhi .............. SUCCESS [  1.809 s]
[INFO] streampipes-processors-transformation-jvm .......... SUCCESS [  4.219 s]
[INFO] streampipes-messaging-jms .......................... SUCCESS [  0.872 s]
[INFO] streampipes-sinks-brokers-jvm ...................... SUCCESS [  1.221 s]
[INFO] streampipes-sinks-databases-jvm .................... SUCCESS [  1.575 s]
[INFO] streampipes-sinks-internal-jvm ..................... SUCCESS [  2.067 s]
[INFO] streampipes-sinks-notifications-jvm ................ SUCCESS [  2.519 s]
[INFO] streampipes-processors-enricher-jvm ................ SUCCESS [  1.696 s]
[INFO] streampipes-processors-change-detection-jvm ........ SUCCESS [  1.130 s]
[INFO] streampipes-extensions-all-iiot .................... SUCCESS [  2.128 s]
[INFO] streampipes-processors-image-processing-jvm ........ SUCCESS [  1.374 s]
[INFO] streampipes-processors-text-mining-jvm ............. SUCCESS [  1.166 s]
[INFO] streampipes-processors-geo-jvm ..................... SUCCESS [  1.690 s]
[INFO] streampipes-extensions-all-jvm ..................... SUCCESS [  3.061 s]
[INFO] streampipes-extensions-iiot-minimal ................ SUCCESS [  1.619 s]
[INFO] StreamPipes Integration Tests ...................... SUCCESS [ 25.671 s]
[INFO] streampipes-rest-core-base ......................... SUCCESS [  1.015 s]
[INFO] StreamPipes REST API ............................... SUCCESS [  2.332 s]
[INFO] streampipes-platform-services ...................... SUCCESS [  1.266 s]
[INFO] streampipes-sdk-bundle ............................. SUCCESS [  0.189 s]
[INFO] StreamPipes Core ................................... SUCCESS [  2.595 s]
[INFO] streampipes-service-core-minimal ................... SUCCESS [  1.187 s]
[INFO] streampipes-wrapper-distributed .................... SUCCESS [  1.071 s]
[INFO] StreamPipes Wrapper for Apache Flink ............... SUCCESS [  1.825 s]
[INFO] streampipes-wrapper-kafka-streams .................. SUCCESS [  1.196 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:50 min
[INFO] Finished at: 2024-04-25T12:48:22+08:00
[INFO] ------------------------------------------------------------------------
paulo@PRED:~/ASF/streampipes$ git status
On branch 2774-fix-TestTextFilterProcessor
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   streampipes-extensions/streampipes-processors-filters-jvm/src/test/java/org/apache/streampipes/processors/filters/jvm/processor/textfilter/TestTextFilterProcessor.java

no changes added to commit (use "git add" and/or "git commit -a")
paulo@PRED:~/ASF/streampipes$ 

```

**Comment by pambrocio** [comment]
PR https://github.com/apache/streampipes/pull/2788

**Comment by tenthe** [comment]
Fixed in PR #2375.
